spec/cc/analyzer/issue_validations/content_validation_spec.rb

```diff
@@ -0,0 +1,31 @@
+require "spec_helper"
+
+module CC::Analyzer::IssueValidations
+  describe ContentValidation do
+    describe "#valid?" do
+      it "is valid without any content" do
+        expect(ContentValidation.new({})).to be_valid
+      end
+
+      it "is valid with content with body" do
+        expect(ContentValidation.new("content" => { "body" => "hi"})).to be_valid
+      end
+
+      it "is not valid with content as string" do
+        expect(ContentValidation.new("content" => "hi")).not_to be_valid
+      end
+
+      it "is not valid with content as nil" do
+        expect(ContentValidation.new("content" => nil)).not_to be_valid
```

Comment:
Got back to this today. For future reference, it seemed like I had to provide an index hint to `count` to get a response in reasonable time.

```
db.runCommand({ 
   count: 'smells',
   query: { _id: { $gt: ObjectId('577144910000000000000000') }, content: { $exists: true, $in: [null] } },
   hint: { _id: 1 }
})

Anyway: no issues in the last day that matched this query, which seems like a large enough data set to me. (That's ~88.7M issues.)
```